pkg/controller/atlasdeployment/advanced_deployment_test.go

```diff
@@ -242,69 +239,3 @@ func TestDbUserBelongsToProjects(t *testing.T) {
 		assert.True(t, dbUserBelongsToProject(dbUser, project))
 	})
 }
-
-func TestAdvancedDeploymentIdle(t *testing.T) {
-	t.Run("should not handle search nodes in Atlas Gov", func(t *testing.T) {
```

Comment:
Why was this test removed?

Reply:
We looked at it for a while, because the migrate broke this test. @s-urbaniak noticed the test is not at the right level of abstraction and can not possibly test what the aim of the test says. It just assumes that calling the idle state and getting an OK workflow means what the test expects, which might not be true.

Testing what this test aimed to test required a refactor or mock outside of the scope of this migration change.